auto_nag/scripts/not_landed.py

```diff
@@ -122,12 +144,14 @@ def check_phab(self, attachment):
             return False
 
         reviewers = data["attachments"]["reviewers"]["reviewers"]
+
         if not reviewers:
             return False
 
         for reviewer in reviewers:
             if reviewer["status"] != "accepted":
                 return False
+            reviewers_phid.add(reviewer["reviewerPHID"])
```

Comment:
The bugzilla username isn't part of the data returned by Phab?

Reply:
No it isn't.